graknlabs/biograkn
# Release CI pipeline phase 1 - structure and assembly

Define BioGrakn CI pipeline skeleton for jobs for master and the release.
Jobs implemented:
- Build
- Test
- Assemble Mocks (excluding bazel test target)
- Assemble All (excluding bazel test target)
- Test Deployment GCP (Check files needed for this job)
- Release approval
- Release cleanup

To be completed:
- Include bazel test targets
- Deploy Github
- Deploy GCP (Check files needed for this job)
- Scheduling of jobs to run
